lib/asyn-ares.c

```diff
@@ -734,7 +734,11 @@ CURLcode Curl_set_dns_servers(struct Curl_easy *data,
     return CURLE_OK;
 
 #if (ARES_VERSION >= 0x010704)
+#if (ARES_VERSION >= 0x011100)
```

Comment:
Ah, I believe this should rather be `>= 0x010b00` since the fields are in hex and 1.11.0 is the version that has this function?

Reply:
that seems likely.